HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs

```diff
@@ -291,16 +291,16 @@ public static Patient JsonToPatient(JSONNode json, string id)
 
             try
             {
-                patient.pid = json["pid"].Value;
-                patient.gender = json["gender"].Value;
-                patient.birthDate = json["birthDate"].Value;
+                patient.pid = json["pid"].Value == "" ? "Unknown" : json["pid"].Value;
```

Comment:
This file is getting a bit long now, maybe you can extract the mapping/data extraction to a different class?